assemblies/plugins/databases/firebird-assemblies/pom.xml

```diff
@@ -10,21 +10,34 @@
         <version>0.1.0-SNAPSHOT</version>
     </parent>
 
-
     <artifactId>hop-assemblies-plugins-databases-firebird</artifactId>
     <version>0.1.0-SNAPSHOT</version>
     <packaging>pom</packaging>
 
     <name>Hop Assemblies Plugins Databases Firebird</name>
     <description></description>
 
+    <properties>        
+        <jaybird.version>2.1.6</jaybird.version>
+    </properties>
 
     <dependencies>
         <dependency>
             <groupId>org.hop</groupId>
             <artifactId>hop-plugins-databases-firebird</artifactId>
             <version>${project.version}</version>
         </dependency>
+    	<dependency>
+      		<groupId>org.firebirdsql.jdbc</groupId>
+      		<artifactId>jaybird</artifactId>
+      		<version>${jaybird.version}</version>
+      		<exclusions>
+       			<exclusion>
+       				<groupId>*</groupId>
+       				<artifactId>*</artifactId>
+      			</exclusion>
+      		</exclusions>
+    	</dependency>        
```

Comment:
We can't include the jaybird driver it is licensed under GPL 2.1 and is incompatible with Apache 2